src/components/Tooltip/Tooltip.spec.tsx

```diff
@@ -3,27 +3,145 @@
 import { mount } from "@cypress/react";
 import React from "react";
 import { Tooltip } from "./Tooltip";
+import IconIcons from "@elements/Icon/Generated/IconIcons";
+import { BrightHeaderStyle } from "./BrightHeader";
 
 const TOOLTIP_TEXT = "This is a tooltip";
-const TOOLTIP_ICON_ID = "[data-test-id=tooltip-icon]";
 const TOOLTIP_ID = "[data-test-id=tooltip]";
+const TOOLTIP_LINK_ID = "[data-test-id=tooltip-link]";
+const TOOLTIP_LINK_URL = "https://www.frontify.com";
+const TOOLTIP_HEADING_TEXT = "I'm a heading";
+const GENERIC_ICON = <IconIcons />;
+const BRIGHT_HEADER_ID = "[data-test-id=bright-header]";
 
 describe("Tooltip Component", () => {
     it("should render a tooltip", () => {
         mount(<Tooltip tooltip={TOOLTIP_TEXT} />);
 
-        cy.get(TOOLTIP_ICON_ID).should("be.visible");
-        cy.get(TOOLTIP_ICON_ID).trigger("mouseover");
-
         cy.get(TOOLTIP_ID).should("contain", TOOLTIP_TEXT);
     });
-    it("should focus with keyboard", () => {
+
+    it("should render an icon next to the tooltip", () => {
+        mount(<Tooltip tooltip={TOOLTIP_TEXT} tooltipIcon={GENERIC_ICON} />);
+
+        cy.get("svg[name=IconIcons]").should("be.visible");
```

Comment:
You could create a reusable variable, e.g. `const SVG_ICON_NAME = "svg[name=IconIcons]"`

Reply:
Alright, I wasn't sure if it was a legit way to check if a specific icon is visible or if there was a more elegant way to do it.